### ISSUE TITLE
Mikrotik: undocumented ui configuration options

### ISSUE THREAD

**Issue by fallingcats** [home-assistant/home-assistant.github.io#24379]
### Feedback

When pressing "configure" in the ui the following options are undocumented:

 - Force scanning using DHCP
 - Enable ARP ping

To me it is unclear
- how the integration detects devices with both checked,
- which metric dhcp detection uses (last seen / lease expiry / both / other?)

### URL

https://www.home-assistant.io/integrations/mikrotik

### Version

2022.9.5

### Additional information

I think those should be documented especially because there is a question mark button one would expect to contain answers but no further explanation is given in the wiki, not even a mention of their existence.

**Comment by homeassistant** [comment]
Hey there @engrbm87, mind taking a look at this feedback as it has been labeled with an integration (`mikrotik`) you are listed as a [code owner](https://github.com/home-assistant/home-assistant.io/blob/current/CODEOWNERS#L375) for? Thanks!

**Comment by pedjas** [comment]
This really needs more detailed info how it works and how to use it.

I set it up and it shows only three network devices as recognized entities although I have 12 of them in a DHCP and few using static addresses. 

Surprisingly, as I have home network separated in two subnets, which HomeAssistant is part just one of, Mikrotik device tracker recognized one device that is in separate subnet, which is good thing.

I've set HA to monitor presence of my mobile phone using Mikrotik device_tracker and found it unreliable. It reacts to device disconnected from the network way to slow and reaction to device connected back is even worse. What exact method is used to detect if device is present or not?
